Parsers/ASimNetworkSession/ProductParsers/NetworkSessionFirewallWindowsEvent.yaml

```diff
@@ -0,0 +1,91 @@
+Parser:
```

Comment:
General comments: 
- We need to add several fields. 
- Most importantly, EventType is absolutely a must. Might be just the default value (NetworkSession I believe). However, what is the difference between the 9 events and how is it reflected in the event? 
- Also, EventResult and DvcAction: I think they are derived from the event IDs. I can also see that some layers are actually failures
- "Listen", which is a layer here, is defined as a direction in my schema.
- Event 5031 is also a firewall event, and an important one.
- Map ProcessId to (Src/Dst)ProcessId
- Map RemoteMachineID to (Src/Dst)HostId

Reply:
1. EventType was present, I changed the value to "NetworkSession"
2.  All Events are: session blocked, session permitted (with slight changes)
3. there is no Failure events -> only blocks/permission from the platform (so, i guess all events are success, from the firewall side)
4. I took it from official Microsoft id to protocol match
5. almost no data in this event (should be represented in all of the above)
6. added!
7. i wansn't sure how to address the other directions ('Forward' & 'Bidirectional'